test/Microsoft.Framework.PackageManager.FunctionalTests/DnuRestoreTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.AspNet.Testing.xunit;
+using Microsoft.Framework.CommonTestUtils;
+using Microsoft.Framework.PackageManager.FunctionalTests;
+using Xunit;
+
+namespace Microsoft.Framework.PackageManager
+{
+    [Collection(nameof(PackageManagerFunctionalTestCollection))]
+    public class DnuRestoreTests
+    {
+        private readonly PackageManagerFunctionalTestFixture _fixture;
+
+        public DnuRestoreTests(PackageManagerFunctionalTestFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        public static IEnumerable<object[]> RuntimeComponents
+        {
+            get
+            {
+                return TestUtils.GetRuntimeComponentsCombinations();
+            }
+        }
+
+        [ConditionalTheory]
+        [MemberData(nameof(RuntimeComponents))]
+        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
```

Comment:
Because i am positive my `bash`-writing skills are out-of-date and don't have a Linux system to play on. We can follow-up later with additional scripts.